### PR TITLE
To Do list: list structure

### PR DESCRIPTION
**Pull request by brainskev**
In this project I was able to accomplish the following;
    Used webpack to bundle JavaScript.
    used proper ES6 syntax.
    Used ES6 modules to write modular JavaScript.

### CODE REVIEW

**Review by omar-muhamad — COMMENTED** [review]
# Invalid ❌

Hi @brainskev,

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

Your Code Review Request is marked as invalid for these reasons:

- [ ] Some linters checks are not passed. Please make sure that all checks are passing and green.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **Does not count** into the code reviews limit._

File: src/style.css [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+  outline: 0.5px black solid;
+}
+
+.todo-description {
+  border: none;
+}
+
+.todo-description:focus {
+  outline: none;
+}
```
Comment (sja-thedude) [inline src/style.css:15]: - [ ] Kindly make the design changes as per the loom video attached.

**Review by sja-thedude — CHANGES_REQUESTED** [review]
## Required Changes ♻️

Hi @brainskev, This is @sja-thedude your code reviewer.

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

768070f3-bc9a-4aba-b8bc-25793e00ea1e_text

## Highlights
✔️ All linters passing
✔️ Professional Readme
✔️ Correct Github Flow

## Required 
- [ ] Kindly make sure to have a look at the loom [video](https://www.loom.com/share/2ec020a4ed91420d9c281559b422edbd) to make changes.
- [ ] _Check the comments under the review._

## Optional suggestions
- [ ] You could use J[S Linters](https://github.com/microverseinc/linters-config/tree/master/javascript) instead of html-css-js linters.

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me [@sja-thedude] in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by sja-thedude** [comment]
Hi @brainskev, While I was reviewing the project you made changes and because of which linters are failing currently. 
When I started with the review linters were passing. Kindly don't commit anything after submitting for review as it confuses the other reviewers and also senior code reviewers at the time of quality checks.

**Review by ha-manel — COMMENTED** [review]
# STATUS: INVALID 🔴 
Hi @brainskev 
I'm sure you put a lot of effort into this project but this PR is invalid because you have failed linters checks 😓 

Kindly fix linters errors before you submit pull requests for review 👍 
Best of luck!

------
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by roshan-bajgain — APPROVED** [review]
Hi @brainskev, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- No linter errors. ✔️
- GitHub flow is followed. ✔️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**